Formula/orientdb.rb

```diff
@@ -64,7 +61,7 @@ def caveats
     sleep 4
 
     begin
-      assert_match "OrientDB Server v.2.2.5", shell_output("curl -I localhost:2480")
+      assert_match "OrientDB Server v.2.2.11", shell_output("curl -I localhost:2480")
```

Comment:
Should I define a version variable ? Where? the cookbook doesn't mention it: https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md

Or  is #{version} implicitly defined in some way? Thanks